releng/platform-definitions/platform-definition-2023-12/platform-definition-2023-12.target

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+   Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Doesn't need to start with the 2023, also you've added the Datadog licensing line in below as a heads up.

Reply:
> Doesn't need to start with the 2023, also you've added the Datadog licensing line in below as a heads up.

Changed the copyright year.